kview/src/main/resources/dev/ikm/komet/kview/mvvm/view/descriptionname/description-name.fxml

```diff
@@ -63,22 +63,22 @@
                               <Insets right="15.0" />
                            </GridPane.margin>
                         </ComboBox>
-                            <ComboBox fx:id="caseSignificanceComboBox" maxWidth="1.7976931348623157E308" prefHeight="25.0" prefWidth="310.0" styleClass="blue-arrow-combo-box" promptText="Select case sensitivity" GridPane.columnIndex="1" GridPane.hgrow="ALWAYS" GridPane.rowIndex="2">
+                            <ComboBox fx:id="caseSignificanceComboBox" maxWidth="1.7976931348623157E308" prefHeight="25.0" prefWidth="310.0" promptText="Select case sensitivity" styleClass="blue-arrow-combo-box" GridPane.columnIndex="1" GridPane.hgrow="ALWAYS" GridPane.rowIndex="2">
```

Comment:
Should the `prompText` be `Select case significance`?

Reply:
Yes, I was looking at wrong FIGMA (concept one).